README.rst

```diff
@@ -29,15 +29,33 @@ Usage:
 
     jinja2_standalone_compiler <path-to-jinja-files>  # Note that jinja files will be searched recursively
 
-For a more intense usage, you can also use a settings file to define a bunch of things. To use settings file:
+* For a more intense usage, you can also use a settings file to define a bunch of things. To use settings file:
 
-.. code:: bash
+    .. code:: bash
+
+        jinja2_standalone_compiler <path-to-jinja-files> --settings=path/to/settings.py
+
+    Please note that settings file is a Python file, so Python syntax applies. =P
```

Comment:
This is being rendered as code on RST as well. Take a look:

![screenshot from 2016-08-31 19-42-21](https://cloud.githubusercontent.com/assets/2631143/18148917/1d61d42e-6fb3-11e6-8d31-299518c755bd.png)